packages/ckeditor5-dev-tests/README.md

```diff
@@ -53,6 +53,7 @@ You can also use the bin script for testing a package:
 * `files` - Specify file(s) to test. Also available as an alias: `-f`.
 * `browsers` - Browsers which will be used to run the tests. Also available as an alias: `-b`.
 * `reporter` - Mocha reporter – either `mocha` (default) or `dots` (less verbose one).
+* `disallow-console-use` - Makes `console.log()/warn()/error()` throw an error when set to `true`
```

Comment:
We're describing CLI flag/option here, so instead of treating it as a bool-ish thing (though in the end it is ofc converted to bool). Actually it would be nice to have this consistent with previous flag, which goes like: "- Whether to (...)".